app-misc/nnn/nnn-4.7.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit bash-completion-r1 flag-o-matic toolchain-funcs xdg-utils
+
+DESCRIPTION="The missing terminal file browser for X"
+HOMEPAGE="https://github.com/jarun/nnn"
+SRC_URI="https://github.com/jarun/nnn/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="+readline pcre 8contexts icons nerdfonts emoji qsort"
+
+DEPEND="sys-libs/ncurses:=
+	pcre? ( dev-libs/libpcre )
+	readline? ( sys-libs/readline:= )
+	elibc_musl? ( sys-libs/fts-standalone )"
+RDEPEND="${DEPEND}"
+REQUIRED_USE="?? ( icons nerdfonts emoji )"
```

Comment:
I think this should be `^^` as in only one of them should be set, because it's perfectly fine to build `nnn` without any icon support. See: https://devmanual.gentoo.org/ebuild-writing/variables/index.html#required_use

```suggestion
REQUIRED_USE="^^ ( icons nerdfonts emoji )"
```

Reply:
Yes, that was my intention. The devmanual indicates that `^^` is "exactly one of" (XOR). `??` is not documented there, but it seems to enforce "at most one of", which is what we need here. Or am I wrong?

Reply:
> Yes, that was my intention. The devmanual indicates that `^^` is "exactly one of" (XOR). `??` is not documented there, but it seems to enforce "at most one of", which is what we need here. Or am I wrong?

I can see `??` documentation just below.

![image](https://user-images.githubusercontent.com/79544946/212592804-d927729e-4b88-4fe6-8fe3-34c7c5abddc4.png)

Also now that I see the docs, I think I'm the one who's messing things up. `??` seems to be the right thing to use here. So ignore my previous comment :)